rockcraft/oci.py

```diff
@@ -349,32 +349,41 @@ def set_entrypoint(self) -> None:
         _config_image(image_path, ["--clear=config.cmd"])
         emit.progress(f"Entrypoint set to {entrypoint}", permanent=True)
 
-    def set_pebble_services(
+    def set_pebble_layer(
         self,
         services: Dict[str, Any],
+        checks: Dict[str, Any],
         name: str,
         tag: str,
         summary: str,
         description: str,
         base_layer_dir: Path,
     ) -> None:
-        """Write the provided services into a Pebble layer in the filesystem.
+        """Write the provided services and checks into a Pebble layer in the filesystem.
 
-        :param services: The Pebble services to be written into the Pebble
+        :param services: The Pebble services
+        :param checks: The Pebble checks
         :param name: The name of the ROCK
         :param tag: The ROCK's image tag
         :param summary: The summary for the Pebble layer
         :param description: The description for the Pebble layer
         :param base_layer_dir: Path to the base layer's root filesystem
         """
         # pylint: disable=too-many-arguments
-        service_names = list(services.keys())
-        emit.progress(f"Configuring Pebble services {', '.join(service_names)}")
         pebble_layer_content = {
             "summary": summary,
             "description": description,
-            "services": services,
-        }
+        }  # type: Dict[str, Any]
```

Comment:
Linters are soon going to start complaining about type comments other than `type: ignore`, so it might be better to say `pebble_layer_content: Dict[str, Any] = {` above.

(I would have put this in as a suggestion but GitHub is being weird about suggestions here.)

Reply:
nice one. will do.